ipn/localapi/localapi.go

```diff
@@ -1904,6 +1905,31 @@ func (h *Handler) serveTKAStatus(w http.ResponseWriter, r *http.Request) {
 	w.Write(j)
 }
 
+func (h *Handler) serveSetGUIVisible(w http.ResponseWriter, r *http.Request) {
+	if !h.PermitWrite {
+		http.Error(w, "GUI visible access denied", http.StatusForbidden)
+		return
+	}
```

Comment:
I'm not sure this bit is required. The SessionID random value is basically the auth.

Reply:
Removed